docs/wmco-prerequisites.md

```diff
@@ -6,11 +6,17 @@ may be relevant.
 | Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
 |---------------------------|---------------------------|------------------------|
 | Amazon Web Services (AWS) | 4.6+                      | WMCO 1.0+              |
-| Azure                     | 4.6+                      | WMCO 1.0+              |
+| Azure                     | 4.11+                     | WMCO 6.0+              |
 | GCP                       | 4.12+                     | WMCO 7.0+              |
 | VMware vSphere            | 4.7+                      | WMCO 2.0+              |
 | Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |
 
+Note: We added Azure support in 4.6 but given Microsoft has [stopped publishing Windows Server 2019 images with
+Docker](https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272),
+we have decided to drop Azure support for releases older than 6.x. From 6.0.0 onwards we are using containerd as the
+runtime and it is WMCO's responsibility to manage that. This is unlike 5.x and below where there was a requirement for
+the Windows Server 2019 images to have Docker pre-installed. 
```

Comment:
Remove:  This is unlike 5.x and below where there was a requirement for
the Windows Server 2019 images to have Docker pre-installed.

Reply:
Done.